src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -151,4 +151,19 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 		snapshot("screenshot_test_certificate")
 	}
 
+	func test_CompleteVaccinationProtectionWithTestCertificate() throws {
```

Comment:
Can you please add the test to the screenshots testplan? Otherwise the screenshot task will not execute this ui"test". Thanks! :)

Reply:
Good to know! Added the other missing one from this file as well :)

Reply:
Thanks! :)

Reply:
Sorry for being bad guy here - Could you also please rename the test name to test_screenshot_CompleteVaccinationProtectionWithTestCertificate for consistency purpose. 

We use this prefix for all the test which involves screenshots.

Reply:
And also for the other missing one as well.

Reply:
You learn something new every day 🤓 Have added it to all the tests in this file, as they are all taking screenshots.

Reply:
Unfortunately that was the only option we were left with to separate screenshots based UI tests.